src/styles/blocks/_grade.scss

```diff
@@ -0,0 +1,13 @@
+.grade {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 16px 0 24px;
+
+  &__text {
+    color: $text-color;
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
Remove it 
```suggestion
```

![image](https://github.com/mate-academy/layout_catalog/assets/70009205/40a3a8da-68ca-4c50-a81b-e379fc0c6bb3)